modules/apps/frontend-js/frontend-js-loader-modules-extender/bnd.bnd

```diff
@@ -1,3 +1,5 @@
 Bundle-Name: Liferay Frontend JS Loader Modules Extender
 Bundle-SymbolicName: com.liferay.frontend.js.loader.modules.extender
-Bundle-Version: 6.0.0
\ No newline at end of file
+Bundle-Version: 6.0.0
+Web-ContextPath: /frontend-js-loader-modules-extender
+-includeresource: META-INF/resources=tmp/META-INF/resources
```

Comment:
This relies on the gradle task that puts some specific dependencies JAR under `tmp`? :thinking:

Reply:
Ah no, it's in the next changeset :point_down: